### DISCUSSION TITLE
I have no need for a "For You" section on my feed

### DISCUSSION THREAD

**Discussion by shaniber** [community/community#13437] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way to disable this? I have no need for an algorithmic discovery feature on my feed. All I need is the updates I requested, in chronological order. 

Thanks,
shane doucette.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
